AUTHORS_RESUBMITTING_UNDER_LGPL_LICENSE.md

```diff
@@ -0,0 +1,43 @@
+# List of authors resubmitting their contributions under LGPL-3.0-or-later
+
+VOLK is going to migrating from GNU General Public License v3.0 or later (GPL-3.0-or-later)
+to GNU Lesser General Public License v3.0 or later (LGPL-3.0-or-later).
+
+This file is a list of the authors who agreed to resubmit their code
+under the LGPL-3.0-or-later license to this repository.
+In case the affected code is licensed differently than VOLK's current license (GPL-3.0-or-later),
+this gives the right to use the current contributions under both LGPL-3.0-or-later and that other license.
+Future contributions by these authors are, however,
+licensed under LGPL-3.0-or-later, unless explicitly stated otherwise.
+
+
+## How to add myself to the table of authors
+There are several options. Choose the one you prefer.
+
+1. Open a PR and add yourself to the table.
+2. Add a Github comment that clearly states your wish to be added to the table.
+3. Write an email to the maintainers that clearly states your wish to be added to the table.
+
+If you go with option 2 or 3, the maintainers add your statement to the corresponding git commit message for documentation purposes.
+
+### Information for option 2 and 3
+
+If you go with option 2 or 3, the maintainers add your statement to the corresponding git commit message for documentation purposes.
+
+Suggested wording
+
+> I, {contributor name}, hereby resubmit all my contributions to the VOLK project and repository under the terms of the LGPLv3+. My GitHub handle is {github handle}, my email addresses used for contributions are: {email address}, ... .
+
+
+## Table of authors
+
+Together with the date of agreement, these authors are:
+
+|       Date | Author (as used in commits) | GitHub handle | Email address(es)        |
+|------------|-----------------------------|---------------|--------------------------|
+| 2021-07-01 | Marcus Müller               |               |                          |
+| 2021-08-01 | Johannes Demel              | jdemel        | <ufcsy@student.kit.edu>, <demel@uni-bremen.de>, <demel@ant.uni-bremen.de> |
+| 2021-08-25 | Martin Braun | mbr0wn        | martin@gnuradio.org, martin.braun@ettus.com  |
+| 2021-08-27 | Carles Fernandez            | carlesfernandez | <carles.fernandez@cttc.es> |
+| 2021-08-27 | Magnus Lundmark | Ka-zam | magnus@skysense.io |
+|            |                             |
```

Comment:
```suggestion
| 2021-09-04 | Michael Dickens | michaelld | mlk@alum.mit.edu, michael.dickens@ettus.com |
|            |                             |
```

Reply:
please don't add more contributor lines to this PR: merge it first, I don't want to become the gatekeeper/bottleneck. Obviously, I'm adding Michael, but he'll be the last one.